requirements/edx/github.txt

```diff
@@ -47,7 +47,6 @@
 # Third-party:
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@v0.0.5#egg=django-wiki==0.0.5
--e git+https://github.com/edx/django-oauth2-provider.git@0.2.7-fork-edx-6a#egg=django-oauth2-provider==0.2.7-fork-edx-6
```

Comment:
edx-django-oauth2-provider (our fork of django-oauth2-provider) is a dependency of edx-oauth2-provider. Confusing, I know.